src/Verify/VerifierSettings.cs

```diff
@@ -30,4 +30,12 @@ public static void AutoVerify(bool includeBuildServer = true)
     }
 
     internal static bool autoVerify;
+
+    public static void UseUtf8NoBom() =>
```

Comment:
is there a test for this?

Reply:
I was concerned about calling the static method in the test since it might impact other tests. Is there a good way for me to isolate this execution? Perhaps it's a solved problem in your test suite.

Reply:
you will need to add a new test project to achieve the isolation

Reply:
Done. Worked locally. Watching the CI.